src/main/java/thePackmaster/cards/creativitypack/Mimicry.java

```diff
@@ -47,7 +47,7 @@ public void update() {
                     }
                     return o;
                 }).collect(Collectors.toCollection(ArrayList::new));
-                addToBot(new FlexibleDiscoveryAction(JediUtil.createCardsForDiscovery(tmpGrp), false));
+                addToTop(new FlexibleDiscoveryAction(JediUtil.createCardsForDiscovery(tmpGrp), false));
```

Comment:
With this change, the card stays up in the background while you're doing the discovery action, which is the same way it works for Wish/Discovery/etc. Need to use addToTop to achieve that due to nested actions.

I'm so used to some aspects of how the base game does things that stuff like this is disconcerting to me if I see it a lot. That's part of how I notice small consistency issues like this.